src/deploy/functions/containerCleaner.ts

```diff
@@ -39,6 +40,25 @@ const SUBDOMAIN_MAPPING: Record<string, string> = {
   "australia-southeast1": "asia",
 };
 
+async function retry<Return>(func: () => Promise<Return>): Promise<Return> {
```

Comment:
I'm a little surprised that we don't have a generic retry library in the CLI at this point (I thought we did), but looks like we don't so 🤷

Reply:
And making it a "real" retry function would involve parameterizing things, so I avoided it in my rush.